pkg/utils/feature_flag.go

```diff
@@ -21,10 +22,31 @@ import (
 	"github.com/pingcap/tidb-dashboard/pkg/utils/version"
 )
 
+type FeatureFlag struct {
+	Name string
+
+	supportedVersions []string
+	supported         *bool
```

Comment:
I think it is not a good idea to make a util stateful.

Reply:
Agreed!